kubernetes/helm_charts/core/nginx-public-ingress/templates/daemonset.yaml

```diff
@@ -5,7 +5,8 @@ metadata:
   name: nginx-public-ingress
   namespace: {{ .Values.namespace }}
   annotations:
-    reloader.stakater.com/auto: "true"
+    # Restarting nginx if ssl cert is changed
+    secret.reloader.stakater.com/reload: "ingress-cert"
```

Comment:
Please add merge domain secrets also

Reply:
Removing the entire block. Nginx reload will take care of the ssl refresh also